.gitignore

```diff
@@ -3,6 +3,7 @@ node_modules
 /dist
 coverage
 playground/*.map
+playground/dist/*
```

Comment:
Please, remove this line. We do need to track some of the files inside `playground/dist` (e.g. `index.html`). Besides, this change should be made in another PR.

Reply:
Right ! My bad